crates/bevy_ui/src/focus.rs

```diff
@@ -52,6 +53,25 @@ impl Default for Interaction {
     }
 }
 
+/// A component storing the position of the mouse relative to the node, (0., 0.) being the upper-left corner and (1., 1.) being the bottom-right
```

Comment:
```suggestion
/// A component storing the position of the mouse relative to the node, (0., 0.) being the upper-left corner and (1., 1.) being the bottom-right
///
```

Reply:
The line break after the basic description serves as a hint to IDEs about what to display on hover :)